scripts/link-liveblocks.sh

```diff
@@ -180,6 +180,9 @@ prep_liveblocks_deps () {
 
             rebuild_if_needed
 
+            # Built package is located in the lib folder, including the package.json
+            cd "./lib"
```

Comment:
This may not be true always, so I'd make this conditional. If a `lib/package.json` exists, then use that, otherwise don't `cd`. This will make it work with packages that don't use a specially crafted
package.json for distribution, or older packages. Can add this in a commit!

Reply:
Added in 18d278a3b438d24ec21d1abdbc499ca4f8b9797a!